win32ss/user/ntuser/message.c

```diff
@@ -1479,7 +1479,7 @@ co_IntSendMessageTimeoutSingle( HWND hWnd,
 
     if (!(Window = UserGetWindowObject(hWnd)))
     {
-        TRACE("SendMessageTimeoutSingle: Invalid handle 0x%p!\n",hWnd);
+        ERR("SendMessageTimeoutSingle: Invalid handle 0x%p!\n",hWnd);
```

Comment:
since you touched the line anyway, I would recommend to strip the final exclamation mark also.